database/database.go

```diff
@@ -154,6 +154,28 @@ func NewCustomDB(ctx context.Context, uri string, dbName string, creds options.C
 	return cdb, nil
 }
 
+// BlockedHashes allows to pass a skip and limit parameter and returns an array
+// of blocked hashes alongside a boolean that indicates whether there's more
+// documents after the current 'page'.
+func (db *DB) BlockedHashes(sort, skip, limit int) ([]BlockedSkylink, bool, error) {
+	// configure the options
+	opts := options.Find()
+	opts.SetSkip(int64(skip))
+	opts.SetLimit(int64(limit + 1))
```

Comment:
Why the `+1`? `db.getCollection('emails').find({}).limit(1)` returns a single record.

Somewhat connected, `return docs[:limit], true, nil` should be `return docs[:limit+1], true, nil`

Reply:
added a comment, the `+1` is just to "peek" and see whether there's more documents after the requested offset